metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql

```diff
@@ -1980,7 +1980,8 @@ SELECT
   FROM_UNIXTIME(END_TIME) as END_TIME,
   END_TIME-START_TIME as ELAPSED,
   ERROR_MESSAGE,
-  FROM_UNIXTIME(LAST_UPDATE_TIME) AS LAST_UPDATE_TIME
+  FROM_UNIXTIME(LAST_UPDATE_TIME) AS LAST_UPDATE_TIME,
+  START_TIME as START_TIME_SINCE_EPOCH
 FROM
```

Comment:
There is upgrade-4.0.0-alpha1-to-4.0.0-alpha-2.hive.sql, too.

Reply:
@pudidic please check now